<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fire</class>
 <widget class="QDialog" name="fire">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>火灾检测</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4,1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="openImageBtn">
       <property name="text">
        <string>打开图片</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openVideoBtn">
       <property name="text">
        <string>打开视频</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openCameraBtn">
       <property name="text">
        <string>打开摄像头</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="stopBtn">
       <property name="text">
        <string>开始检测</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startBtn">
       <property name="text">
        <string>停止检测</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
